src/components/list/list.scss

```diff
@@ -0,0 +1,125 @@
+@import "variables";
+
+$md-list-side-padding: 16px;
+$md-list-avatar-size: 40px;
+
+/*  Normal list variables */
+$md-list-top-padding: 8px;
+$md-list-font-size: 16px;
+$md-list-secondary-font: 14px;
+$md-list-base-height: 48px;
+$md-list-avatar-height: 56px;
+$md-list-two-line-height: 72px;
```

Comment:
Add comments for the `two-line` / `three-line` concept, as it may not be clear to someone unfamiliar with the spec.